CanHazFunny/CanHazFunny/Program.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace CanHazFunny;
+using System;
 
-class Program
+namespace CanHazFunny
 {
-    static void Main(string[] args)
+    class Program
```

Comment:
```suggestion
    public class Program
```
I think CA1852 is showing up because the class isn't accessible (or the accessibility isn't explicit) so the compiler believes it should be sealed